src/main/java/TaskList.java

```diff
@@ -0,0 +1,67 @@
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class TaskList {
+    public static ArrayList<Task> list;
```

Comment:
The tasklist should not be stored static.
Tasklist should be an object.
Causes unit testing to more difficult.